### DISCUSSION TITLE
Setting Chromium Command Line Switches

### DISCUSSION THREAD

**Discussion by aditya-anupam-igt** [Voltstro-Studios/UnityWebBrowser#301] [Q&A]
Hi I'm looking to try a few performance and memory optimizations. Is there a way to set chromium command line switches (e.g., --single-process, --disable-gpu-vysnc) for the browser?

**Comment by Voltstro** [comment]
If you want to add additional flags, you would need to modify the code of UWB.

Also --single-process no longer exists in Chromium.
